docs/documentation/aurora-config.md

```diff
@@ -149,6 +149,8 @@ and _app_ files will be describe in a section called "Application files".
 | permissions/adminServiceAccount |          |              | No           | The service accounts in OpenShift that will have the admin role for the given project. Can either be an array or a space delimited string. This option must be specified in either global or env file.                                                                         |
 | globalFile                      | No       | about.yaml   | globalFile   | Replaces the global file of the project. Note that the default file is the _global_ about file. This option can only be specified in either the _base_ file or _env_ file.                                                                                                     |
 | dataclassification              | No       |              | No           | Sets data classification on the namespace, based on the sensitivity level of the data present in the environment. The value can be set to either synt, skarp, or anon and cannot be changed once set. If it needs to be changed, assistance from an administrator is required. |
+| resourcequota/pv/size           | No       |              | No           | Updates maximum storage size in ResourceQuota for PersistentVolumeClaims. Requires that resourcequota/pv/count is specified.                                                                                                                                                   |
```

Comment:
Lurer på om "Updates" på disse to feltene kan byttes ut med "Set the" eller kun "The" så det blir likere ordbruken andre steder. Vi har ikke brukt update/updates til slik beskrivelse tidligere.

Reply:
enig

Reply:
pushet